mono/utils/mono-threads.h

```diff
@@ -200,7 +200,7 @@ typedef struct _MonoThreadInfo {
 	/*Tells if this thread was created by the runtime or not.*/
 	gboolean runtime_thread;
 
-	/* Max stack bounds, all valid addresses must be between [stack_start_limit, stack_end[ */
+	/* Max stack bounds, all valid addresses must be between [stack_start_limit, stack_end) */
```

Comment:
If we are going to touch this line, then maybe we should use either ( ) or [ ], right now we ended up with [ )

Reply:
`[)` is the correct mathematical notation, for inclusive at the start, exclusive at the end.

Reply:
My edit is specifically fixing the comment.

Reply:
Ah, OK, missed that part.